schemas/org.gnome.shell.extensions.paperwm.gschema.xml

```diff
@@ -217,11 +217,21 @@
       <summary>Cycle through useful window widths</summary>
     </key>
 
+    <key type="as" name="cycle-width-backwards">
+      <default><![CDATA[['<Super><Alt>r']]]></default>
```

Comment:
I think shift might be a better modifier for reversing direction, for consistency with other keybinds

Reply:
Yeah, interestingly, in my tests I changed mine to shift+alt+e (to match my alt+e for widths).

Reply:
I agree -- I intended to use shift as direction modifier, but did not want to change the defaults. Currently, Super+Shift+R is the default keybindings for cycle heights...

What you folks think, should we change these defaults and use shift as backwards and alt for height?

Reply:
Ah - I reckon let's change that sucker (cycle heights), it should probably get it's own key anyways.

I like consistency, and I use (not that it matters here) `alt+e` for cyle width and `alt+q` for cycle heights, and the backwards versions are just appended by `shift`.

Any thoughts on what to change height to?  `alt+t`?

Reply:
I would avoid using `Alt+$LETTER` as default for anything to avoid clashing with app-specific keybindings (not that I use `transpose-words` that often in emacs, but I'd rather not lose arbitrary keybindings when default change ;) )

I had initially though of
- `<Super>+r` and `<Super>+<Shift>+r` for cycle width
- `<Super>+<Alt>+r and <Super>+<Alt>+<Shift>+r` for cycle heights

On using `r` and `t` because they are side-by-side: Currently, `<Super>+t` is `take window`, so `<Super>+R` and `<Super>+T` for the cycle would require changing the default for take -- and IMHO the current default is good.

On using `w` and `h` for _width_ and _height_: `<Super>+w` would clash with the Switcher extension and `<Super>+h` would annoy weird people like me who like to use vim-like `<Super>+hjkl` movements.

Reply:
> Any thoughts on what to change height to? `alt+t`?

Oops - meant to say `<Super>+t` - but as you said that's taken.

Reply:
> and IMHO the current default is good.

Agree - unless there's a good alternative to cycle heights, happy to leave it as it currently is.